src/main/java/redis/clients/jedis/resps/TrackingInfo.java

```diff
@@ -0,0 +1,70 @@
+package redis.clients.jedis.resps;
+
+import redis.clients.jedis.Builder;
+import redis.clients.jedis.util.KeyValue;
+
+import java.util.Collections;
+import java.util.List;
+
+import static redis.clients.jedis.BuilderFactory.*;
+
+public class TrackingInfo {
+
+    private final List<String> flags;
+    private final long redirect;
+    private final List<String> prefixes;
+
+    public TrackingInfo(List<String> flags, long redirect, List<String> prefixes) {
+        this.flags = flags;
+        this.redirect = redirect;
+        this.prefixes = prefixes;
+    }
+
+
+    public List<String> getFlags() {
+        return flags;
+    }
+
+    public long getRedirect() {
+        return redirect;
+    }
+
+    public List<String> getPrefixes() {
+        return prefixes;
+    }
+
+    public static final Builder<TrackingInfo> TRACKING_INFO_BUILDER = new Builder<TrackingInfo>() {
+        @Override
+        public TrackingInfo build(Object data) {
+            List commandData = (List) data;
+
+            if (commandData.get(0) instanceof KeyValue) {
```

Comment:
https://github.com/redis/jedis/pull/3751#pullrequestreview-1926556648
This one has also been completed btw with its test [here](https://github.com/redis/jedis/pull/3751/commits/bac0296e5f294a34f1dadb5d86b856278674c38c#diff-5482ee92f736fc02b8cad49e22343bfabffaf25fc3a32a3ba512ee1c979e22b6R295)